contrib/README.rst

```diff
@@ -0,0 +1,38 @@
+labgrid-webapp
+==============
+
+labgrid-webapp implements a browser interface to access some of labgrid's
+information.
+
```

Comment:
Should the `graphviz` command line tool requirement be mentioned?

Reply:
the `graphviz` library is listed in the `requirements-webapp.txt` file, with some instructions on how to get things running. would you want to see an explicit mention of the library? I'm not sure it's really useful, but we could add it.

Reply:
Providing the requirements file made running the application so simple I had forgotten about the `graphviz` Python library. My thought was explicitly calling out the system dependency may help future users.

Reply:
I might be missing something, but I don't think there's a dependency on the `graphviz` system package. Did you have to install it to get things working? I might've had it installed already.

Reply:
https://graphviz.readthedocs.io/en/stable/manual.html does mention needing an installation of Graphviz. I got a very lengthy error from the `graphviz` Python library when I tried viewing the graph web page before I installed Graphviz.

Reply:
Oh! thanks for pointing that out. I added a note in the README.